packages/leancode_lint/lib/helpers.dart

```diff
@@ -218,6 +217,92 @@ extension LintRuleNodeRegistryExtensions on LintRuleNodeRegistry {
       listener(buildMethod.body, diagnosticNode);
     });
   }
+
+  void addBloc(void Function(ClassDeclaration node, _BlocData data) listener) {
```

Comment:
We could add some documentation here. AddBloc is, if I am not wrong, limited to one file. So _BlocData data wouldn't be correct fully. So there is an assumption that the whole bloc with events, states, is in one file. It would be worth mentioning this.